### PR TITLE
Adding document for setting introduced : cluster.remote_store.translo…

### PR DESCRIPTION
**Pull request by astute-decipher**
…g.max_readers

### Description
We introduced a new remote cluster setting as a part of https://github.com/opensearch-project/OpenSearch/pull/12992

### Issues Resolved
Closes [#7077](https://github.com/opensearch-project/documentation-website/issues/7077)

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thanks, @astute-decipher ! Let me know when you are good with this from a technical perspective so we can begin doc review.

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 1]
```diff
@@ -62,11 +62,13 @@ During the bootstrapping process, the remote-backed repositories listed in `open
...
+| cluster.minimum.index.refresh_interval        | Time unit | Sets the minimum refresh interval and applies it to all indexes in the cluster. The `cluster.default.index.refresh_interval` setting should be higher than this setting's value. If, during index creation, the `index.refresh_interval` setting is lower than the minimum, index creation fails.                                       |
+| cluster.remote_store.translog.buffer_interval | Time unit | The default value of the translog buffer interval used when performing periodic translog updates. This setting is only effective when the index setting `index.remote_store.translog.buffer_interval` is not present.                                                                                                                   |
+| cluster.remote_store.translog.max_readers     | Integer   | Sets the maximum number of open translog files for remote backed indexes. This setting is introduced to limit the total number of translog files per shard. Upon breaching which, flush will be triggered and translog files will be trimmed.                                                                                           |
```
Comment (BhumikaSaini-Amazon) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:70]:
```suggestion
| cluster.remote_store.translog.max_readers     | Integer   | Sets the maximum number of open translog files for remote-backed indices. This setting limits the total number of translog files per shard. Upon breaching this limit, a flush will be triggered and the translog files will be trimmed. Default is `1000` and the minimum required is `100`.                                                                                          |
```

Added the default and minimum values enforced. Please adjust the above for formatting, if needed.

**Review by BhumikaSaini-Amazon — CHANGES_REQUESTED** [review]

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 2]
```diff
@@ -62,11 +62,13 @@ During the bootstrapping process, the remote-backed repositories listed in `open
...
+| cluster.default.index.refresh_interval        | Time unit | Sets the refresh interval when the `index.refresh_interval` setting is not provided. This setting can be useful when you want to set a default refresh interval across all indexes in a cluster and also support the `searchIdle` setting. You cannot set the interval lower than the `cluster.minimum.index.refresh_interval` setting. |
+| cluster.minimum.index.refresh_interval        | Time unit | Sets the minimum refresh interval and applies it to all indexes in the cluster. The `cluster.default.index.refresh_interval` setting should be higher than this setting's value. If, during index creation, the `index.refresh_interval` setting is lower than the minimum, index creation fails.                                       |
+| cluster.remote_store.translog.buffer_interval | Time unit | The default value of the translog buffer interval used when performing periodic translog updates. This setting is only effective when the index setting `index.remote_store.translog.buffer_interval` is not present.                                                                                                                   |
```
Comment (ashking94) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:69]: Can you undo the changes done in these lines?

**Review by ashking94 — CHANGES_REQUESTED** [review]
Lets remove the unnecessary changes done.

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md (hunk 2 above)
Comment (astute-decipher) [reply _tuning-your-cluster/availability-and-recovery/remote-store/index.md]: Done

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md (hunk 1 above)
Comment (astute-decipher) [reply _tuning-your-cluster/availability-and-recovery/remote-store/index.md]: Sure, added these changes.

**Review by BhumikaSaini-Amazon — APPROVED** [review]

**Review by ashking94 — APPROVED** [review]

**Comment by astute-decipher** [comment]
> Thanks, @astute-decipher ! Let me know when you are good with this from a technical perspective so we can begin doc review.

Hi @hdhalter, we are done with technical review and good to merge, let me know if there are any comments from your side. Thanks.

File: _tuning-your-cluster/availability-and-recovery/remote-store/index.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ You can use the following [cluster settings]({{site.url}}{{site.baseurl}}//api-r
...
 | cluster.minimum.index.refresh_interval | Time unit | Sets the minimum refresh interval and applies it to all indexes in the cluster. The `cluster.default.index.refresh_interval` setting should be higher than this setting's value. If, during index creation, the `index.refresh_interval` setting is lower than the minimum, index creation fails. |
 | cluster.remote_store.translog.buffer_interval | Time unit | The default value of the translog buffer interval used when performing periodic translog updates. This setting is only effective when the index setting `index.remote_store.translog.buffer_interval` is not present. |
-
+| cluster.remote_store.translog.max_readers | Integer | Sets the maximum number of open translog files for remote-backed indexes. This limits the total number of translog files per shard. Upon breaching this limit, a flush will be triggered and the translog files will be trimmed. Default is `1000` and the minimum required is `100`.|
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/availability-and-recovery/remote-store/index.md:70]:
```suggestion
| cluster.remote_store.translog.max_readers | Integer | Sets the maximum number of open translog files for remote-backed indexes. This limits the total number of translog files per shard. After reaching this limit, the remote-store flushes the translog files. Default is `1000`. The minimum required is `100`. |
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

**Review by Naarcha-AWS — APPROVED** [review]

**Review by Naarcha-AWS — APPROVED** [review]
